src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java

```diff
@@ -71,6 +71,7 @@ public class AllChecksTest extends AbstractModuleTestSupport {
         new Locale("tr"),
         Locale.CHINESE,
         Locale.ENGLISH,
+        new Locale("ru"),
```

Comment:
Please put it alphabet order, as you did at `name="requiredTranslations`

Reply:
Done
I sorted all locales
In this file too - config/checkstyle_checks.xml